spec/views/recipe_foods/create.html.erb_spec.rb

```diff
@@ -0,0 +1,3 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
```

Comment:
- [ ] This is a redundant test file, kindly take it out or write a view test in it